ui/decredmaterial/theme.go

```diff
@@ -75,6 +75,8 @@ type Theme struct {
 
 	Clipboard     chan string
 	ReadClipboard chan interface{}
+
+	dropDownMenus []*DropDown
```

Comment:
I don't think closing drop down menus should be done through out the entire app. The dropdowns that are related can be in a group and only the dropdowns in the group will be closed when one of the items is selected.